src/solver/simulation/solver.hxx

```diff
@@ -1,1636 +1,1644 @@
-                                                   y,
-                                                   yearsIndicesCopy,
-                                                   set_it->yearFailed,
-                                                   set_it->isFirstPerformedYearOfASet,
-                                                   pFirstSetParallelWasRun,
-                                                   numSpace,
-                                                   randomForParallelYears,
-                                                   performCalculations,
-                                                   study,
-                                                   state,
-                                                   pYearByYear));
-
-        } // End loop over years of the current set of parallel years
-
-        qs.start();
-
-        qs.wait(Yuni::qseIdle);
-        qs.stop();
-
-        // At this point, the first set of parallel year(s) was run
-        if (!pFirstSetParallelWasRun)
-            pFirstSetParallelWasRun = true;
-
-        // On regarde si au moins une ann�e du lot n'a pas trouv� de solution
-        std::map<uint, bool>::iterator it;
-        bool foundFailure = false;
-        for (it = set_it->yearFailed.begin(); it != set_it->yearFailed.end(); it++)
-        {
-            if (it->second)
-            {
-                foundFailure = true;
-                break;
-            }
-        }
-        // Si une ann�e du lot d'ann�es n'a pas trouv� de solution, on arr�te tout
-        if (foundFailure)
-        {
-            logs.fatal() << "At least one year has failed in the previous set of parallel year.";
-            AntaresSolverEmergencyShutdown();
-        }
-
-        // Computing the summary : adding the contribution of MC years
-        // previously computed in parallel
-        ImplementationType::variables.computeSummary(set_it->spaceToPerformedYear,
-                                                     set_it->nbPerformedYears);
-
-        // Computing summary of spatial aggregations
-        ImplementationType::variables.computeSpatialAggregatesSummary(
-          ImplementationType::variables, set_it->spaceToPerformedYear, set_it->nbPerformedYears);
-
-        // Computes statistics on annual (system and solution) costs, to be printed in output into
-        // separate files
-        if (not study.parameters.adequacyDraft())
-            computeAnnualCostsStatistics(state, set_it);
-
-        // Set to zero the random numbers of all parallel years
-        randomForParallelYears.reset();
-
-    } // End loop over sets of parallel years
-
-    // Writing annual costs statistics
-    if (not study.parameters.adequacyDraft())
-    {
-        pAnnualCostsStatistics.endStandardDeviations();
-        pAnnualCostsStatistics.writeToOutput();
-    }
-}
-
-} // namespace Simulation
-} // namespace Solver
-} // namespace Antares
-
-#endif // __SOLVER_SIMULATION_SOLVER_HXX__
+/*

```

Comment:
There was no change in this file. See if clang was passed.

Reply:
I passed dos2unix on that file, this solves the problem